teamcity/messages.py

```diff
@@ -10,6 +10,14 @@
     text_type = str
 
 
+quote = {"'": "|'", "|": "||", "\n": "|n", "\r": "|r", ']': '|]'}
```

Comment:
Changed `quote to `_quote`. The missing `[` is a good question for @shalupov I guess; as this was existing code that I simply moved from inside the `TeamcityServiceMessages` class to the global scope.